crates/backend/src/codegen.rs

```diff
@@ -136,6 +138,17 @@ impl TryToTokens for ast::Program {
     }
 }
 
+fn gen_id() -> [u8; 8] {
+    use ::std::time::{SystemTime, UNIX_EPOCH};
+
+    let duration = SystemTime::now().duration_since(UNIX_EPOCH).unwrap();
+    
+    let mut bytes = duration.as_secs().to_be_bytes();
+    bytes[0..4].copy_from_slice(&duration.subsec_nanos().to_be_bytes());
```

Comment:
I think that the CI server is failing due to identifier collisions that are generated here: I don't see these when building locally, presumably because my machine is much slower and/or much less parallel than Azure such that it can generate two separate IDs in the same nanosecond whereas I cannot?  Or perhaps [`Duration::subsec_nanos`](https://doc.rust-lang.org/std/time/struct.Duration.html#method.subsec_nanos) is returning 0 on Windows?

Reply:
Turns out that this is precisely what the issue that `backend::util::ShortHash` was built to resolve, however I still witnessed collisions when building `web_sys`: adding a timestamp to each package's (cached) hash seed solved that, but raises the question of whether the seed ought not just be entirely random?